features/leaveroom/api/src/main/res/values/localazy.xml

```diff
@@ -1,5 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
+  <string name="leave_conversation_alert_empty_subtitle">"Are you sure that you want to leave this conversation? You\'re the only person here. If you leave, no one will be able to join in the future, including you."</string>
+  <string name="leave_conversation_alert_private_subtitle">"Are you sure that you want to leave this conversation? This conversation is not public and you won\'t be able to rejoin without an invite."</string>
+  <string name="leave_conversation_alert_subtitle">"Are you sure that you want to leave the conversation?"</string>
```

Comment:
I do not think that a user can see this string, since I guess all the DMs are not public.
So maybe for DM the application should ignore the public/private state of the room?

I just check, and currently, when leaving a DM, the string `leave_room_alert_private_subtitle` is displayed.

Reply:
Yeah, you're right. A DM will only be considered a DM when it's direct and has 2 users, so the `private` text here will always be triggered. The other two aren't needed. Thanks!

Reply:
OK, So I think only the string `leave_conversation_alert_subtitle` should be used in this case.

Reply:
I'd actually use the contents of `leave_conversation_alert_private_subtitle`:

> Are you sure that you want to leave this conversation? This conversation is not public and you won't be able to rejoin without an invite.

It seems the most accurate one, given the situation.

Reply:
Or maybe removing the 'This conversation is not public and" part, just saying "You won't be able to rejoin without an invite", I don't know.

Reply:
> It seems the most accurate one, given the situation.

I agree, but also the most complex, and we tend to simplify the user life in the context of Element X. That's why a product POV may be helpful. CC @VolkerJunginger